.github/workflows/tests.yaml

```diff
@@ -0,0 +1,47 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
I don't think this is an issue but want to double check: if joe schmoe opens a PR against this repo, can they do stuff like make changes to this workflow that logs api keys or anything like that? I'm pretty sure the answer is no